students/lauraannf/Lesson09/run_mailroom_oo.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov 29 11:51:56 2018
+
+@author: Laura.Fiorentino
+"""
+
+
+from mailroom_oo import *
```

Comment:
Recommend simply import mailroom_oo

Reply:
you can do 
`from mailroom_oo import send_thank_you`

however, since you're using many functions it is better to do:
```
import mailroom_oo


def test_send_thank_you():
    mailroom_oo.send_thank_you()  # access via module
```